src/NodeVisitor/EscaperNodeVisitor.php

```diff
@@ -183,12 +183,11 @@ private function isSafeFor(string $type, Node $expression, Environment $env): bo
         return \in_array($type, $safe) || \in_array('all', $safe);
     }
 
-    private function needEscaping(Environment $env)
```

Comment:
Parameter not used since [`1f00c33`](https://github.com/twigphp/Twig/commit/1f00c33acde77c1c3eb5c2c13c292f5e5afcffda#diff-f94a61cb3a67823e5fe2dd545a52b47962d3486d8eb9919106c69dd7910d12b5L138)